src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -213,10 +214,14 @@ private void loadNextEventAnimation()
       GameEvent event = eventsToAnimate.peek();
       currentAnimation = event.getEventAnimation( this );
       boolean isEventHidden = !(null == event.getStartPoint()) && gameMap.isLocationFogged(event.getStartPoint()) && gameMap.isLocationFogged(event.getEndPoint());
-      if( null == currentAnimation || isEventHidden || drawFogEverywhere )
+      if( null != currentAnimation && (isEventHidden || drawFogEverywhere) )
```

Comment:
Why not keep the condition the same, and just always at least animate nothing?

Reply:
Didn't wanna break anything accidentally by changing the function behavior too much.

Reply:
Yeah, but at the same time, we could easily still have a chain of un-animated events that cause the same overflow error. Since the most common actions currently have animations, that's not super likely, but I'd rather just moot the possibility.
I'd really prefer to not even need a non-animation, but that might be a dream for a future day.